Sources/SwiftSyntaxBuilder/gyb_generated/BuildableNodes.swift

```diff
@@ -11043,13 +10795,13 @@ public struct CatchClause: SyntaxBuildable, ExpressibleAsCatchClause {
   ///  - Initializing tokens without default text using strings
   public init(
     catchKeyword: TokenSyntax = TokenSyntax.`catch`,
-    body: ExpressibleAsCodeBlock,
-    @CatchItemListBuilder catchItemsBuilder: () -> ExpressibleAsCatchItemList? = { nil }
```

Comment:
Please keep this as `CatchItem` conforms to `HasTrailingComma`

Reply:
❌ Similar to `if`. The body is the primary body here.